lib/iris/fileformats/pp_rules.py

```diff
@@ -1058,7 +1062,17 @@ def _all_other_rules(f):
             cell_methods.append(CellMethod(method,
                                            coords='time'))
 
-    if f.lbproc not in [0, 128, 4096, 8192]:
+    if zonemethod is not None:
```

Comment:
I'm not the biggest fan of the zonemethod variable. How about simply checking the lbproc again here. ``if lbproc == 192`` is equivalent at this point.